src/foam/i18n/TranslationService.js

```diff
@@ -18,9 +18,26 @@ foam.INTERFACE({
     {
       name: 'getTranslation',
       async: true,
-      args: [ 'String locale', 'String source' ],
-//      javaType: 'foam.i18n.Locale'
-type: 'String'
+      args: [
```

Comment:
If you change the interface then you will also need to update the ClientCacheTranslationService implemenation.

Reply:
@kgrgreer implementation or parameters for the method on the client side?